trulens_eval/examples/frameworks/custom/custom_example.ipynb

```diff
@@ -85,10 +85,10 @@
    "metadata": {},
```

Comment:
<p>Can you explain a bit more why this custom class example is useful?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/425/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
accidentally changed from full repo string replace